common/config/rush/command-line.json

```diff
@@ -67,6 +91,14 @@
       "shellCommand": "rush update -p --variant stable --full && rush update -p --full",
       "safeForSimultaneousRushProcesses": true
     },
+    {
```

Comment:
it may be usefult to add a command for `rush update:stable` too if we have one for "rush update"

Reply:
Updated